src/components/CommandBar.tsx

```
@@ -175,16 +179,16 @@ export function CommandBar() {
                   <CommandItem
                     key={setting.id}
                     onSelect={() => {
-                      setSetting(setting.id, !getSetting(setting.id));
+                      setSetting(setting.id as any, !getSetting(setting.id as any));
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)